store/rootmulti/store.go

```diff
@@ -378,23 +380,46 @@ func (rs *Store) TracingEnabled() bool {
 	return rs.traceWriter != nil
 }
 
-// AddListeners adds listeners for a specific KVStore
-func (rs *Store) AddListeners(key types.StoreKey, listeners []types.WriteListener) {
-	if ls, ok := rs.listeners[key]; ok {
-		rs.listeners[key] = append(ls, listeners...)
-	} else {
-		rs.listeners[key] = listeners
+// AddListeners adds a listener for the KVStore belonging to the provided StoreKey
+func (rs *Store) AddListeners(keys []types.StoreKey) {
```

Comment:
I'm reluctant to change that in our fork. I would like to make the suggestion upstream because it's already been accepted as `AddListeners` in the ADR proposal so it's going to require a discussion.

Reply:
Is is acceptable?

Reply:
Sounds good to me.